shm/inc/alcohol.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "cargo.hpp"
+
+namespace {
+constexpr auto kBasePower = 96.f;
+}
+
+class Alcohol : public Cargo {
+public:
+    Alcohol(const std::string& name, size_t amount, size_t basePrice);
+    Alcohol(const std::string& name, size_t amount, size_t basePrice, size_t power);
+    ~Alcohol() override = default;

```

Comment:
Do we have any rule about moving constructors to cpp even when they are trivial?

Reply:
Yup. Mateusz said that in our review from part one